### ISSUE TITLE
twtxt2rss ?

### ISSUE THREAD

**Issue by sorenpeter** [jointwt/rss2twtxt#9]
Is there an easy way to take a twtxt and get out a rss feed?
I guess there is something in https://github.com/jointwt/twtxt/ but i would not know where to start diggin

**Comment by prologic** [comment]
On any user's profile page there is a link to their Atom feed which is exaactly the same as their Twtxt feed.

<img width="1312" alt="Screen Shot 2021-01-21 at 08 39 42" src="https://user-images.githubusercontent.com/1290234/105249702-438d8d80-5bc4-11eb-92ef-fb5ce6e52b37.png">

For example mine are:

* https://twtxt.net/user/prologic/twtxt.txt
* https://twtxt.net/user/prologic/atom.xml

**Comment by sorenpeter** [comment]
Thank, i know. that is how i have it set up my twtxt.net feed on my own website: www.darch.dk/microblog 
What I'm looking for is a way to the same with my self-hosted file, so I can feed it into this setup or others can get an rss  feed into their reader of choice.

**Comment by prologic** [comment]
So you effectively want some way to go from twtxt to RSS or Atom given an external twtxt feed?

**Comment by sorenpeter** [comment]
Yes. Maybe this was not the place to ask about it. I have not found something that seem to be able to do it. But I guess you got it worked out since it is a thing at twtxt.net

**Comment by prologic** [comment]
> Yes. Maybe this was not the place to ask about it. I have not found something that seem to be able to do it. But I guess you got it worked out since it is a thing at twtxt.net

Ahh I see what you're trying to do.

Yeah, here's what I would do.

- Pick a library that knows how to construct Atom.
- Read it's documentation (_hopefully it has examples!_)
- Parse your twtxt.txt file and construct your Atom ndoes
- Render this

That's basically what the `twtd` backend does for the `AggregationHandler` (_loook at the source_)
